js/controller.js

```diff
@@ -417,10 +417,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         // time to show is based on percentage of duration from the beginning
         timeToShow = (1 - parseInt(stringTimeToShow)/100) * duration;
       }
-
       this.state.upNextInfo.timeToShow = timeToShow;
 
-      if (duration - currentPlayhead <= timeToShow &&
+       if (duration!==0 && duration- currentPlayhead <= timeToShow &&
```

Comment:
- remove extra space in front of `if`
- add spaces around operator, should be `duration !== 0`
- keep each condition on separate line